resources.md

```diff
@@ -12,7 +12,8 @@ Anyone with a ualberta account can access the below links. You do need
 to be logged in with that username, to see these google drive files.
 
 <ul>
- 	<li><a href="https://docs.google.com/document/d/1bQRRNhffZi5hUzeWU18qAf0dyN1zKBYIZk1o9-5r1FI/">How to write a MSc thesis</a></li>
+    <li><a href="https://drive.google.com/drive/u/1/folders/0B18wGbj9Q3U4N3NyZDMtWk5PODg">RLAI Google Drive</a></li>
```

Comment:
Would you mind dropping the `/u/1` portion of the link? This specifies a google account index if you've logged into drive with multiple accounts, and is very person specific. Without it in the link, google will either open with the default account or will ask which account to use (which is useful in this case since only `@ualberta.ca` accounts have access)

Reply:
Thanks for letting me know. I will send another PR.

Reply:
Already pushed. Do I need to send another PR?

Reply:
Nope, new commits to this branch are automatically added to the PR.

Thanks!